config/bin_version_strings_links.cfg

```diff
@@ -0,0 +1,39 @@
+# EMBA - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2020-2021 Siemens Energy AG
+# Copyright 2020-2021 Siemens AG
+#
+# EMBA comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# EMBA is licensed under GPLv3
+#
+# Author(s): Michael Messner, Pascal Eckmann
+#
+# Description: Version database for detection of binary versions via emulation and static analysis
+#
+# example:
+# license:link to license
```

Comment:
could you change the deliminiter to ;
I'm currently also working on this on the bin_versions configuration here: https://github.com/m-1-k-3/emba/commit/d7c56768ce91e779290633a677e8f97d92d5898c

Reply:
Done (and tested -> works).